src/controller_initializer.coffee

```diff
@@ -4,10 +4,12 @@ fs = require 'fs'
 path = require 'path'
 
 
-class ControllerStore
+# Initializes controllers in @controllersDir and supports calling saved controller actions
+class ControllerInitializer
 
   constructor: (@controllersDir, @dependencies) ->
     @controllers = {}
+    @initialize()
```

Comment:
A better name for this method seems `loadControllers` or `initializeControllers`